File: src/main/java/com/example/demo/service/UserService.java
package com.example.demo.service;

import com.example.demo.entity.UserBean;
import com.example.demo.impl.UserMapper;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class UserService implements UserMapper {

    @Autowired
    UserMapper userMapper;


    @Override
    public UserBean login(String phone, String password) {
        return userMapper.login(phone, password);
    }

    @Override
    public void register(String phone, String password) {
        userMapper.register(phone, password);
    }

    @Override
    public UserBean findUser(String phone) {

        return userMapper.findUser(phone);
    }


}

File: src/main/java/com/example/demo/service/UserService.kt
package com.example.demo.service

import com.example.demo.entity.UserBean
import com.example.demo.impl.UserMapper
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

/**
 * @Author Mr.Wu
 * @Date create on 2020/1/11
 * @DES
 */
@Service
class UserService : UserMapper {
    @Autowired
    private val userMapper: UserMapper? = null

    override fun login(phone: String?, password: String?): UserBean? = userMapper!!.login(phone, password)

    override fun register(phone: String?, password: String?) = userMapper!!.register(phone, password)

    override fun findUser(phone: String?): UserBean? = userMapper!!.findUser(phone)

}